tests/hp/solution_transfer_03.cc

```diff
@@ -91,7 +91,7 @@ main()
   data_out.write_vtu(deallog.get_file_stream());
 
 
-  // Interpoalte solution
+  // Interpolate solution
```

Comment:
Nice. Thank you for making these small changes too while you're there -- it is appreciated!